cuda/test_wrapper_thapilog.sh.in

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+: "${CC:=gcc}"
+
+btx_model_path="@builddir@/btx_cuda_model.yaml"
+trace_path="$1"
+trace_name="$(basename $trace_path)"
+trace_name="${trace_name%.thapilog}"
```

Comment:
does `(basename $trace_path%.thapilog)` work?

Reply:
Oh actually it does! I tried basename inside `${...%...}` and that did not work, but basename can be done after as you suggest.